ui/pages/Project/components/FamilyTable/CaseReviewStatusDropdown.jsx

```diff
@@ -5,9 +5,10 @@ import PropTypes from 'prop-types'
 import styled from 'styled-components'
 import { connect } from 'react-redux'
 
-import ReduxFormWrapper from 'shared/components/form/ReduxFormWrapper'
+import StateChangeForm from 'shared/components/form/StateChangeForm'
```

Comment:
`ReduxFormWrapper` holds  A LOT of state -  its designed to have users edit different fields and then hit submit and do something with all the form data, so the backing form library is keeping all the form field data in the state, including validating inputs and keeping validation state in the state. Its also designed to dispatch an action when the form submits and the render differently depending on whether it is currently submitting or whether submission has succeeded/ failed, which is more state. Also, its designed to be used in a modal, so it maintains the state of if the modal should be ope or closed. This is all really helpful because almost every form we use in seqr has this behavior so having this helper class that manages all that state is great. 

However, there are some places where we are using a "form" that doesn't do ANY of that. Its just a bunch of inline inputs that dispatch an acton for each separate input whenever that specific input changes. Theres no case where you ever need the state of the whole "form" input at once (no submit button at all), no validation, no success/ failure callbacks for submissions, no modal. So setting up and listening to all that state actually adds a bunch of unnecessary overhead. Therefore, I made a lightweight helper `StateChangeForm` that will render inputs that look the same as the other form class, but without setting up any of the state